.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,163 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  Core:
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Core
+      run: cmake --build build -j 2 --target netmeld-core
+
+    - name: Test Core
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-core
+        cd build
+        ctest -L ^Test.netmeld-core$
+    
+    - name: Package Core
+      run: |
+        cd ./scripts
+        bash ./make_netmeld-core_deb.sh
+        cd ..
+    
+    - name: Upload Deb Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: netmeld-core-deb
+        path: scripts/*.deb
+  
+  
+  CoreModules:
+    needs: [Core]
+    
+    runs-on: ubuntu-latest
+    container: netmeld/netmeld-dev:latest
+
+    strategy:
+      matrix:
+        core-modules: [playbook-core, datalake-core]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure All
+      run: cmake -S . -B build
+
+    - name: Build Core Modules
+      run: cmake --build build -j 2 --target netmeld-${{ matrix.core-modules }}
+
+    - name: Test Core Modules
+      run: |
+        cmake --build build -j 2 --target Test.netmeld-${{ matrix.core-modules }}
+        cd build
+        ctest -L ^Test.netmeld-${{ matrix.core-modules }}$
+    
+    - name: Package Core Modules
+      if: matrix.core-modules != 'datalake-core'
```

Comment:
Is this because we don't have a build script for the Datalake module?

Reply:
yes, those if-checks can be removed once we have those scripts in place. Otherwise the CI workflow fails and doesn't finish all the other jobs.